config/job_config_sample.yaml

```diff
@@ -52,6 +52,9 @@ variables:
  TrimmedDateFormat: yyyy/MM/dd
 
 output:
+  # elasticsearch Database argument: nodes is mandatory.
+  elasticsearch:
+    nodes: localhost
```

Comment:
Don't you want to couple the node hostnames/IPs with ports? maybe even make it an array?

Reply:
Sure, good idea. But why pass an array? isn't host/port enough?

Reply:
Done